spec/pardot/objects/prospects_spec.rb

```diff
@@ -55,4 +55,47 @@ def sample_results
       assert_authorization_header
     end
   end
+
+  context 'for API version 4:' do
```

Comment:
I guess batch is only supported in 4?

I noticed that by default the client is instantiated as v3, we should probably fix that in our app.

Reply:
I believe that batch is supported in v4.